package.json

```diff
@@ -30,6 +30,7 @@
   },
   "scripts": {
     "build": "buidler compile && tsc",
+    "prepare": "yarn prepack",
```

Comment:
Who will be calling `yarn prepare`? Is it automatically the first option git dependencies if provided?

Reply:
Yarn will be calling it when installing this as a dependency in another project.